system-variables.md

```diff
@@ -61,6 +61,12 @@ SET  GLOBAL tidb_distsql_scan_concurrency = 10;
 - Default value: 0
 - This variable is used to show whether the execution plan used in the previous `execute` statement is taken directly from the plan cache.
 
+### last_plan_from_binding <span class="version-mark">New in v4.0</span>
+
+- Scope: SESSION
+- Default value: 0
+- This variable is used to show whether the execution plan used in the previous `execute` statement is from the plan binding.
```

Comment:
```suggestion
- This variable is used to show whether the execution plan used in the previous executed statement is from the plan binding.
```

Reply:
I have two corrections:
- It does not apply to the previous execute statement, but the previously executed statement (this was probably copied from `last_plan_in_cache` incorrectly).
- It is not the plan binding, but a plan binding (since there can be multiple different bindings).

May I suggest: "This variable is used to show whether the execution plan used in the previous statement was influenced [by a plan binding](/sql-plan-management.md)".

Reply:
Thanks! I change my comment as you suggested.